prov/verbs/src/fi_verbs.c

```diff
@@ -252,11 +260,20 @@ int vrb_get_rai_id(const char *node, const char *service, uint64_t flags,
 	int ret;
 
 	// TODO create a similar function that won't require pruning ib_rai
-	ret = vrb_get_rdma_rai(node, service, flags, hints, rai);
+	if (hints) {
+                ret = vrb_get_rdma_rai(node, service, hints->addr_format, hints->src_addr,
+					hints->src_addrlen, hints->dest_addr,
+					hints->dest_addrlen, flags, rai);
+	} else {
+                ret = vrb_get_rdma_rai(node, service, FI_FORMAT_UNSPEC, NULL, 0, NULL,
+				       0, flags, rai);
+        }
```

Comment:
Please change all spaces to tabs at the beginning of lines (except for spaces at the end of blank spaces to align function parameters).